Dangerfile.swift

```diff
@@ -1,4 +0,0 @@
-import Danger
-import WeTransferPRLinter
-
-WeTransferPRLinter.lint()
```

Comment:
Why did we remove this?

Reply:
Our CI [improved lately](https://github.com/WeTransfer/WeTransfer-iOS-CI/pull/117) which allows us to reuse this file as well from CI.